modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticServiceMetrics.java

```diff
@@ -191,6 +197,31 @@ private CachedMeters metersForEndpoint(String endpoint) {
            Optional.empty();
   }
 
+  /**
+   * Checks for latency-slo-metrics configuration options and sets up
+   * metrics that track how many requests meet a certain latency goal.
+   *
+   * @param id
+   * @param endpoint
+   * @return
+   */
+  private Optional<LatencySloTracker> requestLatencySloTracker(MetricId id, String endpoint) {
+    final String[] endpointComponents = endpoint.split(":");
+    final String key = String.format("latency-slo-metrics.%s", endpointComponents[1],
+        endpointComponents[0]);
+    final String allKey = "latency-slo-metrics.all-endpoints";
```

Comment:
Looks like there's not tests covering `latency-slo-metrics.all-endpoints` and `latency-slo-metrics.SOME_ENDPOINT`.